.github/workflows/deploy_gh_packages.yml

```
@@ -0,0 +1,55 @@
+# This workflow will build and deploy the application
+name: Deploy to GH Packages
+
+env:
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.PERSONAL_TOKEN }}
+  NODE_VERSION: "14.x"
+  SEMANTIC_RELEASE_VERSION: 17.1.1
```

could be `18.0.0`, which is the latest
```suggestion
  SEMANTIC_RELEASE_VERSION: v18.0.0
```